### DISCUSSION TITLE
stella with other than adafruit feather?

### DISCUSSION THREAD

**Discussion by JuaquinWalling** [STELLA-Landsat/STELLA#45] [Build Notes: How do I make a STELLA?]
is there a way to build the stella 2 using another micro controller, since the adafruit board seems to be unavailable?
i'm trying to build a stella 2 for the local planetarium to use for outreach.  ,

**Comment by paulmirel** [comment]
It's possible to swap in a different controller, but only with different software. We don't have alternative software available for the STELLA-2. 

Mouser has that microcontroller in stock. https://mou.sr/3FaxiGY

Thanks for your enthusiasm and interest in STELLA!

**Reply by JuaquinWalling** [reply]
It brings Landsat down to earth, to a level we can all understand.

**Comment by JuaquinWalling** [comment]
Thank you for the response.  I'll order one from Mouser and get on with
building a Stella 2.  Time to finally get my 3D printer to work and build
the stl parts.

I downloaded the code and looked at porting it to a Raspberry pi pico w,
but got stuck on the libraries.

I'll send you a picture when the planetarium gets its Stella.

Thanks again for getting back to me, and doing it in the first place.  It
will be a wonderful teaching tool and inspire kids to learn because, as you
said, they can actually do sensor work with their own hands.
